docs/griptape-framework/drivers/vector-store-drivers.md

```diff
@@ -406,3 +406,112 @@ vector_store_driver.upsert_text_artifacts(
 result = vector_store_driver.query("What is griptape?")
 print(result)
 ```
+
+### Qdrant
+
+!!! info
+    This driver requires the `drivers-vector-qdrant` [extra](../index.md#extras).
+
+The QdrantVectorStoreDriver supports the [Qdrant vector database](https://qdrant.tech/).
+
+Here is an example of how the driver can be used to query information in a Qdrant collection:
+
+```python
+import os
+import logging
+from transformers import AutoTokenizer
+from sentence_transformers import SentenceTransformer
+from griptape.drivers import QdrantVectorStoreDriver, HuggingFaceHubEmbeddingDriver
+from griptape.tokenizers import HuggingFaceTokenizer
+from griptape.loaders import PdfLoader
+from griptape.chunkers import TextChunker
+
+# Configure logging
+logging.basicConfig(level=logging.INFO)
+
+# Setting the models
+embedding_model = "sentence-transformers/all-MiniLM-L6-v2"
+qdrant_model = SentenceTransformer("all-MiniLM-L6-v2")
+file_name = "linux_bible.pdf"
+HUGGINGFACE_TOKEN = os.environ["HUGGINGFACE_HUB_ACCESS_TOKEN"]
+
+
+# Using HuggingFaceTokenizer
+def create_tokenizer(embedding_model):
+    tokenizer = HuggingFaceTokenizer(max_output_tokens=1024, tokenizer=AutoTokenizer.from_pretrained(embedding_model))
+    return tokenizer
+
+
+def create_embedding_driver(embedding_model, tokenizer):
+    embedding_driver = HuggingFaceHubEmbeddingDriver(
+        api_token=HUGGINGFACE_TOKEN, model=embedding_model, tokenizer=tokenizer
+    )
+    return embedding_driver
+
+
+# Instantiating QdrantVectorStoreDriver
+def create_vector_store_driver(url, collection_name, embedding_driver):
+    vector_store_driver = QdrantVectorStoreDriver(
+        url=url, collection_name=collection_name, content_payload_key="content", embedding_driver=embedding_driver
```

Comment:
pass an api_key here using:
```
os.environ["QDRANT_CLUSTER_API_KEY"]
```
and please add the same variable [in this file](https://github.com/griptape-ai/griptape/blob/dev/.github/workflows/docs-integration-tests.yml). Following the same convention as the others, where it will be referenced as `INTEG_QDRANT_CLUSTER_API_KEY`